api/src/main/java/org/openmrs/api/impl/MedicationDispenseServiceImpl.java

```diff
@@ -9,19 +9,16 @@
  */
 package org.openmrs.api.impl;
 
-import org.apache.commons.lang.BooleanUtils;
+import java.util.List;
+
 import org.apache.commons.lang3.StringUtils;
 import org.openmrs.MedicationDispense;
 import org.openmrs.api.APIException;
 import org.openmrs.api.MedicationDispenseService;
-import org.openmrs.api.context.Context;
 import org.openmrs.api.db.MedicationDispenseDAO;
 import org.openmrs.parameter.MedicationDispenseCriteria;
 import org.springframework.transaction.annotation.Transactional;
 
-import java.util.Date;
-import java.util.List;
```

Comment:
Are these changes part of the ticket requirement?

Reply:
The ticket requirement required an update of apache velocity, in my update some of the packages were nolonger required hence removing them because they are no used. Infact  my IDE automatically removes unused packages.